src/main/scala/logphy/LogicalPhy.scala

```diff
@@ -0,0 +1,17 @@
+package edu.berkeley.cs.ucie.digital.logphy
+
+import chisel3._
+import edu.berkeley.cs.ucie.digital.interfaces._
+
+class LogicalPhy(
+    lanes: Int = 16,
+    serializerRatio: Int = 16,
+    width: Int,
+    sbWidth: Int,
```

Comment:
We may want to pull these configuration knobs out into a separate case object, so that we don't duplicate defaults at every point that they are used.